src/chromo/kinematics.py

```diff
@@ -230,11 +236,44 @@ def __init__(
 
         _gamma_cm = (elab + m2) / ecm
         _betagamma_cm = plab / ecm
+        self.m1 = m1
```

Comment:
Add a comment above as doc, that that may introduce some E/p conservation issue if the internal mass in the model is too different to that from `particle`/PDG/Pythia.

Reply:
The comment is added to the code of the beam definition.